web/src/upload.js

```diff
@@ -281,7 +281,7 @@ export function setup_upload(config) {
         },
     );
 
-    $("#edit_form_banners").on(
+    $(".edit_form_banners").on(
```

Comment:
This fixes the fact that this was broken, but this whole construction is wrong; @evykassirer FYI as well, since this was introduced in .b07bc23e07ceec55ba5768c82dd8303587906521

Every time we open a message edit form, the existing logic creates a new click handler for close buttons in compose banners. 

There's two sane ways to do things -- only create a click handler for the element specified in the config, or do a single global handler defined in an `initialize` method and registered on body. Neither of these is doing that.

Reply:
Ah whoops, thanks for catching this!